Makefile

```diff
@@ -41,7 +58,11 @@ docker/weave:
 	curl -L git.io/weave -o docker/weave
 	chmod u+x docker/weave
 
-$(SCOPE_EXPORT): $(SCOPE_EXE) $(DOCKER_DISTRIB) docker/weave $(RUNSVINIT) docker/Dockerfile docker/demo.json docker/run-app docker/run-probe docker/entrypoint.sh
+docker/ebpf.tgz: Makefile
+	$(SUDO) docker pull $(EBPF_IMAGE)
+	CONTAINER_ID=$(shell $(SUDO) docker run -d $(EBPF_IMAGE) /bin/false 2>/dev/null || true); $(SUDO) docker export -o docker/ebpf.tgz $${CONTAINER_ID}
```

Comment:
I am also adding logs in CircleCI to make it easier to figure out which version of ebpf.o it is using.

![image](https://cloud.githubusercontent.com/assets/27575/22249992/899285a8-e245-11e6-9cdd-a272632cae7e.png)

Also, I am adding that info in the Scope Docker image to make it easier to debug:

![image](https://cloud.githubusercontent.com/assets/27575/22249619/07279bf4-e244-11e6-8481-48c7ee0acdcd.png)